_ingest-pipelines/processors/index-processors.md

```diff
@@ -69,6 +69,12 @@ Processor type | Description
 `urldecode` | Decodes a string from URL-encoded format.
 `user_agent` | Extracts details from the user agent sent by a browser to its web requests. 
 
+## Validations on Processors
```

Comment:
Can we call it as "Processor limit settings"?

Reply:
Updated.